.github/workflows/fpga.yml

```diff
@@ -25,6 +25,9 @@ on:
       rom-version:
         default: "latest"
         type: string
+      rom-ref:
```

Comment:
I think ultimately our goal is for the nightly release workflow to be runnable with specified ROM & FW refs. Would it make sense to try to factor out some of the steps in this file so it can be reused in the nightly one as well?

Not sure if that would improve things or just make the CI more confusing.

Reply:
One thing is still unclear to me: We want to build a release (probably using the `build_release.sh` script) for specific revisions of FMC, ROM, and RT.
You also said it should be integrated into the nightly release flow iirc.

So what we can do is to set the default values to HEAD and main and if desired we can change that. How about the testing? As of know, all the test suites are run as part of the nightly release, which makes sense. Is this also a requirement for the specifically 
versioned checkouts or should we speed things up by only packaging the release?

Reply:
When you say you want to have this ultimately being run in the nightly workflow I think I don't fully understand what you mean by that.
The FPGA workflows are all pinned to versions, or do you mean we should add another test case that is run for these said specified versions that might be passed as inputs to the nightly workflow? Or should we just consider building the release using the `build_release.sh` script for these versions?